source/preprocessor.tex

```diff
@@ -1665,8 +1665,9 @@
 The macros defined in \tref{cpp.predefined.ft} shall be defined to
 the corresponding integer literal.
 \begin{note}
-Future revisions of \Cpp{} might replace
-the values of these macros with greater values.
+Future revisions of \Cpp{} will replace
+the values of these macros with greater values
+when the corresponding feature is modified.
```

Comment:
This sounds like an established statement of intent, but it's one that has not been approved by the committee as far as I'm aware.

Reply:
Agreed, reverted.